src/util/resize.js

```diff
@@ -0,0 +1,50 @@
+export const DIVIDER_HEIGHT = 4;
+export const DEFAULT_VERTICAL_FLEX = [
+  `1 1 ${DIVIDER_HEIGHT}px`,
+  `1 1 ${DIVIDER_HEIGHT}px`,
```

Comment:
I would think the default vertical flex for all the editor containers would still just be `0 1 100vh`—this seems like another thing we could keep in CSS, though? I.e. use the stylesheet flex unless there has been a resize, and only apply a `style` attribute if we need to override the default